packages/components/src/components/tabs/tab.jsx

```diff
@@ -19,10 +19,12 @@ const Tab = ({
     is_active,
     is_scrollable,
     is_label_hidden,
+    is_p2p,
```

Comment:
Please don't use package specific props in other packages. Please use a reusable prop name.

Reply:
What if it is a package specific case?